orchagent/muxorch.cpp

```diff
@@ -534,6 +534,11 @@ bool MuxCable::nbrHandler(bool enable, bool update_rt)
 void MuxCable::updateNeighbor(NextHopKey nh, bool add)
 {
     sai_object_id_t tnh = mux_orch_->getNextHopTunnelId(MUX_TUNNEL, peer_ip4_);
+    if (add && skip_neighbors_.count(nh.ip_address) != 0)
```

Comment:
Please use find instead of count as the intend is to check if the key exists. This is the current pattern in the file.